src/parser/print_visitor.rs

```diff
@@ -52,6 +52,7 @@ impl WhammVisitor<String> for AsStrVisitor {
     fn visit_whamm(&mut self, whamm: &Whamm) -> String {
         let mut s = "".to_string();
 
+        // Alex: arne't these fn? why call them events
```

Comment:
Oops, that was a refactoring issue. Dtrace uses the name `function` for what we call `event`, so when I refactored to `event`, I just did this one on accident. If you see more things like this, feel free to switch the name if you think it makes sense!